klaytnetl/service/klaytn_contract_service.py

```diff
@@ -45,9 +45,17 @@ def get_function_sighashes(self, bytecode):
                 push4_instructions = [
                     inst for inst in instructions if inst.name == "PUSH4"
                 ]
```

Comment:
Do we need to make another for loop for `PUSH3`? How about merging to find `PUSH3` and `PUSH4`